frontend/src/DiagramEditor/ContextMenu/ContextMenu.scss

```diff
@@ -53,7 +53,7 @@
     }
 
     &:hover {
-      background-color: #fbfbfb;
+      background-color: #f5f5f5;
```

Comment:
shall we do a media query to check if they have hover at all?

Reply:
the whole concept right now is based on using a mouse. We have context menus, which are triggered with right click, and they are central to the app

Reply:
I see! we can make an ipad version later ;)

Reply:
yep, that would be awesome, but it would have different mechanics